messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/BridgeAdd.java

```diff
@@ -173,41 +175,55 @@ static BridgeConfiguration createBridgeConfiguration(final OperationContext cont
         return result;
     }
 
-    static void createBridge(String name, BridgeConfiguration bridgeConfig, ActiveMQServerControl serverControl) {
+    static void createBridge(BridgeConfiguration bridgeConfig, ActiveMQServer server) {
+        checkStarted(server);
+        clearIO(server);
+
         try {
-            String transformerClassName = bridgeConfig.getTransformerConfiguration() != null ? bridgeConfig.getTransformerConfiguration().getClassName() : null;
-            if (bridgeConfig.getDiscoveryGroupName() != null) {
```

Comment:
Why is this bridgeConfig settings no longer needed because of adding "call-timeout" attribute?

Reply:
I'm gonna close this PR for now, this was the version from before it was recommended to modify Artemis API first.

Reply:
The BridgeConfig instance will now be directly passed to `ActiveMQServer#deployBridge(BridgeConfig)`, rather than the attributes being unpacked from BridgeConfig and passed to `ActiveMQServerControl#createBridge(String, String, String, ...)` explicitely.

(In the original impl, the `ActiveMQServerControlImpl#createBridge(String, String, String, ...)` then reconstructs the `BridgeConfig` instance and passes it to `ActiveMQServer#deployBridge(BridgeConfig)` anyway, so we are just taking a shortcut here.)

Not sure if this was what you were asking though :).

Reply:
Thanks!